sdk/azidentity/README.md

```diff
@@ -59,132 +51,122 @@ to authenticate requests.
+- [Authenticate with DefaultAzureCredential](#authenticate-with-defaultazurecredential "Authenticate with DefaultAzureCredential")
+- [Define a custom authentication flow with ChainedTokenCredential](#define-a-custom-authentication-flow-with-chainedtokencredential "Define a custom authentication flow with ChainedTokenCredential")
 
-### Authenticating with `DefaultAzureCredential`
-This example demonstrates authenticating the `ResourcesClient` from the armresources module using `DefaultAzureCredential`.
+### Authenticate with DefaultAzureCredential
+This example demonstrates authenticating a client from the `armresources` module with `DefaultAzureCredential`.
 
 ```go
-// The default credential checks environment variables for configuration.
 cred, err := azidentity.NewDefaultAzureCredential(nil)
 if err != nil {
   // handle error
 }
 
-// Azure SDK Azure Resource Management clients accept the credential as a parameter
-client := armresources.NewResourcesClient(armcore.NewDefaultConnection(cred, nil), "<subscription ID>")
+client := armresources.NewResourceGroupsClient("subscription ID", cred, nil)
 ```
 
 See more how to configure the `DefaultAzureCredential` on your workstation or Azure in [Configure DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-go/wiki/Set-up-Your-Environment-for-Authentication#configure-defaultazurecredential).
 
-### Authenticating a user assigned managed identity with `DefaultAzureCredential`
-This example demonstrates authenticating the `ResourcesClient` from the armresources module using the `DefaultAzureCredential`, deployed to an Azure resource with a user assigned managed identity configured.
+#### Specify a user assigned managed identity for `DefaultAzureCredential`
+To configure `DefaultAzureCredential` to authenticate a user assigned managed identity, set the
+environment variable `AZURE_CLIENT_ID` to the identity's client ID.
+
+### Define a custom authentication flow with `ChainedTokenCredential`
+
+`DefaultAzureCredential` is generally the quickest way to get started developing
+applications for Azure. For more advanced scenarios,
+[ChainedTokenCredential][chain_cred_ref] links multiple credential instances
+to be tried sequentially when authenticating. It will try each chained
+credential in turn until one provides a token or fails to authenticate due to
+an error.
 
-See more about how to configure a user assigned managed identity for an Azure resource in [Enable managed identity for Azure resources](https://github.com/Azure/azure-sdk-for-go/wiki/Set-up-Your-Environment-for-Authentication#enable-managed-identity-for-azure-resources).
+The following example demonstrates creating a credential which will attempt to
+authenticate using managed identity, and fall back to authenticating via the
+Azure CLI when a managed identity is unavailable.
```

Comment:
```suggestion
The following example demonstrates creating a credential, which will attempt to authenticate using managed identity. It will fall back to authenticating via the Azure CLI when a managed identity is unavailable.
```